training_pipeline/models/hyperparams.py

```diff
@@ -1,10 +1,6 @@
 INPUT_IMG_SIZE = 128
 
-TRAIN_LENGTH = 800
-EVAL_LENGTH = 200
-
 TRAIN_BATCH_SIZE = 64
 EVAL_BATCH_SIZE = 64
 
-EPOCHS = 30
-LR = 0.001
```

Comment:
why omit LR ?

Reply:
I am not trying to match the hyperparameters followed by the TensorFlow tutorial. They use the defaults.